scripts/linters/pylint_extensions.py

```diff
@@ -1751,40 +1751,34 @@ class NewlineBelowClassDocstring(checkers.BaseChecker):
     }
 
     def visit_classdef(self, node):
-        """Process a module to ensure that there is a single newline below
-        class docstring.
+        """Visit each class definition in a module.
 
         Args:
-            node: astroid.scoped_nodes.Function. Node to access module content.
+            node: astroid.scoped_nodes.ClassDef. Node for a class definition
```

Comment:
I think it should be nodes not scoped_nodes, can you please check? (not sure!)

Reply:
I checked it is scoped_nodes

Reply:
Can you please add a reference? (I'm writing doc for pylint extensions and I'll use this link there!)

Reply:
Sorry, my mistake, I've checked it from older implemented checks, but after checking the official documentation I found that it is node. http://pylint.pycqa.org/projects/astroid/en/latest/api/astroid.nodes.html?highlight=visit_classdef#astroid.nodes.ClassDef

Reply:
Thanks for the link, can you please update the type here? :)

Reply:
Done.